libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3725,9 +3725,12 @@ void GCS_MAVLINK::handle_common_message(const mavlink_message_t &msg)
         handle_set_mode(msg);
         break;
 
+#if AP_MAVLINK_AUTOPILOT_VERSION_REQUEST_ENABLED
     case MAVLINK_MSG_ID_AUTOPILOT_VERSION_REQUEST:
+        send_received_message_deprecation_warning("AUTOPILOT_VERSION_REQUEST");
```

Comment:
this needs to be changed from WARNING to INFO

Reply:
This is already done in master